backend/tests/contract/test_contract_master.py

```
@@ -0,0 +1,167 @@
+]
+
+
+def _is_excluded(case) -> bool:
+    """Return True if (method, path) is on the day-1 exclusion list."""
+    return (case.method.upper(), case.path) in DAY_1_EXCLUSIONS
+
+
+@pytest.mark.integration
+@contract_settings
+@schema.parametrize()
+def test_contract_endpoint_returns_documented_shape(case, client):
```

Ruff (0.15.13)

[warning] Unused function argument: `client`

(ARG001)